### PR TITLE
Prepare void return while SHUTDOWN

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

File: src/main/java/redis/clients/jedis/BinaryJedis.java [hunk 1]
```diff
@@ -3470,18 +3471,21 @@ public Long lastsave() {
...
+  @Deprecated
+  public String shutdown() throws JedisException {
     client.shutdown();
-    String status;
     try {
-      status = client.getStatusCodeReply();
-    } catch (JedisException ex) {
-      status = null;
+      String status = client.getStatusCodeReply();
+      throw new JedisException(status);
```
Comment (gkorland) [inline src/main/java/redis/clients/jedis/BinaryJedis.java:3484]: @sazzad16 won't it always going to throw a JedisException?

Comment (gkorland) [inline src/main/java/redis/clients/jedis/BinaryJedis.java:3484]: @sazzad16 I get that you want to deprecate it, but why do you change the logic?

Comment (sazzad16) [reply src/main/java/redis/clients/jedis/BinaryJedis.java:3484]: @gkorland I've added throwing exception when there's an error. Other than that, the logic is same. E.g. `null` is returned when everything goes well both in existing code and in new code.

**Comment by sazzad16** [comment]
Closing in favor of #2613

**Pull request closed without merging**